pkg/cmd/server/origin/master.go

```diff
@@ -309,11 +355,17 @@ func (c *MasterConfig) Run(protected []APIInstaller, unprotected []APIInstaller)
 
 	handler = open
 
+	handler = indexAPIPaths(handler)
```

Comment:
Yes for indexAPIPaths.  assetServerRedirect needs to come after the call to ensureCORSAllowedOrigins, it didnt seem right to add CORS headers when we are going to be sending a redirect to the web console